frontend_vue/src/components/tokens/kubeconfig/TokenDisplay.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div class="flex justify-center">
+    <base-button
+      class="mt-16"
+      @click="handleDownloadKubeconfigFile"
+      >Download your tokened Kubeconfig file</base-button
+    >
+  </div>
+</template>
+<script setup lang="ts">
+import { downloadAsset } from '@/api/main';
+
+type PDFDataType = {
```

Comment:
PDF datatype?

Reply:
woopsy, left over from the copy/paste!
I change it, thank you!